sdk/src/asset_handlers/bmff_io.rs

```diff
@@ -1257,4 +1386,25 @@ pub mod tests {
         }
         assert!(success)
     }
+
+    #[test]
+    fn test_remove_c2pa() {
+        let source = fixture_path("video1.mp4");
+
+        if let Ok(temp_dir) = tempdir() {
```

Comment:
Since you're in test code, why not use `.unwrap()` rather than the nested `if let`s?

Reply:
Some intermediate version of Clippy made me do it.  If necessary I can see if it is still an issue.